felix/dataplane/windows/endpoint_mgr.go

```diff
@@ -191,7 +191,8 @@ func (m *endpointManager) RefreshHnsEndpointCache(forceRefresh bool) error {
 		// Some CNI plugins do not clear endpoint properly when a pod has been torn down.
 		// In that case, it is possible Felix sees multiple endpoints with the same IP.
 		// We need to filter out inactive endpoints that do not attach to any container.
-		if len(endpoint.SharedContainers) == 0 {
+		// An endpoint is considered to be active if its state is Attached or AttachedSharing.
+		if (endpoint.State.String() != "Attached" && endpoint.State.String() != "AttachedSharing") {
```

Comment:
Our pre-flight check linter is not happy with the parentheses in here
```suggestion
		if endpoint.State.String() != "Attached" && endpoint.State.String() != "AttachedSharing" {
```

Reply:
Thank you for the pointer!